airflow/datasets/manager.py

```diff
@@ -58,6 +64,12 @@ def create_datasets(self, dataset_models: list[DatasetModel], session: Session)
         for dataset_model in dataset_models:
             self.notify_dataset_created(dataset=Dataset(uri=dataset_model.uri, extra=dataset_model.extra))
 
+    def create_dataset_aliases(self, dataset_alias_models: list[DatasetAliasModel], session: Session) -> None:
+        """Create new dataset aliases."""
+        for dataset_alias_model in dataset_alias_models:
+            session.add(dataset_alias_model)
```

Comment:
Can this use `add_all` instead? (And if it does, is a separate function still useful?)

Reply:
Yes, `add_all` sounds much better and we could thus remove this function.

Reply:
Tested and updated